### PR TITLE
Clean up command extensions

### PR DESCRIPTION
**Pull request by nadjaheitmann**
This PR is based on the commits in PR #1
Based on https://github.com/theforeman/hammer-cli-foreman/pull/569

### CODE REVIEW

File: lib/hammer_cli_foreman_puppet.rb [hunk 1]
```diff
@@ -8,36 +8,36 @@ module HammerCLIForemanPuppet
   require 'hammer_cli_foreman_puppet/option_sources'
   require 'hammer_cli_foreman_puppet/associating_commands'
 
-  require 'hammer_cli_foreman_puppet/organization_extenstions'
-  require 'hammer_cli_foreman_puppet/location_extenstions'
+#  require 'hammer_cli_foreman_puppet/organization_extenstions'
+#  require 'hammer_cli_foreman_puppet/location_extenstions'
```
Comment (amirfefer) [inline lib/hammer_cli_foreman_puppet.rb:12]: can you please delete these lines?

File: lib/hammer_cli_foreman_puppet/host.rb [hunk 2]
```diff
@@ -41,7 +40,7 @@ class InfoCommand < HammerCLIForeman::InfoCommand
...
     HammerCLIForemanPuppet::CommandExtensions::PuppetEnvironment.new
   )
-  HammerCLIForeman::Host::CommonUpdateOptions.extend_with(
-    HammerCLIForemanPuppet::CommandExtensions::PuppetEnvironment.new
-  )
-end
\ No newline at end of file
+  #HammerCLIForeman::Host::CommonUpdateOptions.extend_with(
+  #  HammerCLIForemanPuppet::CommandExtensions::PuppetEnvironment.new
+  #)
```
Comment (amirfefer) [inline lib/hammer_cli_foreman_puppet/host.rb:45]: is this reduantant?

**Review by amirfefer — COMMENTED** [review]
Thanks for the cleanup! 

I'm just wondering if we need to add `PP` prefix, if we can test this plugin with https://github.com/theforeman/hammer-cli-foreman/pull/571

File: lib/hammer_cli_foreman_puppet.rb (hunk 1 above)
Comment (nadjaheitmann) [reply lib/hammer_cli_foreman_puppet.rb]: Done.

File: lib/hammer_cli_foreman_puppet/host.rb (hunk 2 above)
Comment (nadjaheitmann) [reply lib/hammer_cli_foreman_puppet/host.rb:45]: As far as I can see, this is not needed as there is no such extension in hammer-cli-foreman. Feel free to double-check.

**Comment by nadjaheitmann** [comment]
> > Thanks for the cleanup!
> > I'm just wondering if we need to add `PP` prefix, if we can test this plugin with [theforeman/hammer-cli-foreman#571](https://github.com/theforeman/hammer-cli-foreman/pull/571)
> 
> IMHO, I would go for it because this is somehow analogue to how things are done in the Puppet extraction in Foreman core. Right now, Foreman works with Puppet plugin and Puppet in Foreman core. API is slightly different for the plugin. So my suggestion is to make the Hammer plugin compatible with the current state of `hammer-cli-foreman` and with the extraction (as long as it does not get more complicated than changing the naming). If there is a good reason for dropping the extension, we can also drop it. I am aware that there is a slight overhead in adding the `pp` prefix.

I have been thinking once again. To make it similar to the Hammer situation in Foreman right now, we would need to overwrite existing Puppet calls in `hammer-cli-foreman` with the ones from Puppet plugin. Seems like a lot of effort to me. So as you and @ofedoren have suggested, maybe we should skip the renaming part and just test with your `puppet_extraction` PR.

So @amirfefer, @ofedoren, if you agree I will make the changes to #1, #2, #3 and we can keep moving forward :)

**Pull request closed without merging**